package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "typedoc",
   "description": "Create api documentations for typescript projects.",
-  "version": "0.14.2",
+  "version": "0.14.3",
```

Comment:
Please don't change the package version number in PRs. This will be done during the release. Thanks!

Reply:
sorry, didn’t know that. want me to make another commit or leave it as it is?

Reply:
No problem, it's something that's not aways clear for contributors. I'll probably make updates to all the dependencies now that a fix has been pushed so don't worry about updating.